### PR TITLE
urls and views tests

### PR DESCRIPTION
**Pull request by radekwlsk**

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            old_settings[k] = settings_module.user_settings[k]
+        except KeyError:
+            pass
+
+        # Install temporary settings
+        settings_module.user_settings[k] = v
+
+        # Delete any cached settings
+        try:
+            delattr(settings_module, k)
```
Comment (MKaras93) [inline tests/utils.py:37]: Can you explain what's going on here? What is `settings_module.k`, when above you are using only `settings_module.user_settings`?

**Review by MKaras93 — APPROVED** [review]
Only one question.

**Comment by radekwlsk** [comment]
With the newest updates we have now 100% coverage:

```
---------- coverage: platform darwin, python 3.7.5-final-0 -----------
Name                                   Stmts   Miss  Cover
----------------------------------------------------------
tenc_graphene/__init__.py                  0      0   100%
tenc_graphene/migrations/__init__.py       0      0   100%
tenc_graphene/schema.py                   33      0   100%
tenc_graphene/settings.py                 11      0   100%
tenc_graphene/urls.py                      4      0   100%
tenc_graphene/views.py                     8      0   100%
----------------------------------------------------------
TOTAL                                     56      0   100%
```

Yay!

File: tests/utils.py (hunk 1 above)
Comment (radekwlsk) [reply tests/utils.py:37]: I guess it is done to remove any default/cached settings as the comment indicates and not from the `user_settings` that is just a kind of storage for them not the settings themselves. Code is taken from https://github.com/davesque/django-rest-framework-simplejwt/blob/master/tests/utils.py which I could have noted but as it is MIT i just did not.

**Pull request merged**